wallet/src/wallet.rs

```diff
@@ -92,11 +91,8 @@ pub struct WalletConfig {
     pub command: WalletCommand,
     pub drone_host: Option<IpAddr>,
     pub drone_port: u16,
-    pub host: IpAddr,
+    pub json_rpc_url: String,
```

Comment:
Move keypair field after json_rpc_url for consistency?

Reply:
✔️